_locales/en/messages.json

```diff
@@ -188,6 +215,9 @@
   "dark": {
     "message": "Dark"
   },
+  "toggleTheme": {
+    "message": "Toggle dark mode"
```

Comment:
It would be nice (and easier to translate) if there were separate "Switch to dark theme" and "Switch to light theme" messages. ("Theme", not "mode", for consistency with the text in More Settings.)